sirepo/sim_data/__init__.py

```diff
@@ -66,10 +66,11 @@ def audit_proprietary_lib_files(qcall, force=False, sim_types=None):
       sim_types (set): Set of sim_types to audit (proprietary_sim_types if None)
     """
     from sirepo import simulation_db
+    import sirepo.sim_run
```

Comment:
imports like this don't work right. Do this:

from sirepo import simulation_db, sim_run

Reply:
fixed with 4aaf2e6

It looks like there are many places in the code which do an import from a method in the bad format:
$ trgrep "   import sirepo"

Reply:
I'm not sure this is still a problem but it was a problem at one point. After some testing it seems to import `sirepo` in the local namespace, which is the right thing to do.